### PR TITLE
Updating board application link

### PR DESCRIPTION
**Pull request by Crystal-Luo**

### CODE REVIEW

**Review by mzhang28 — APPROVED** [review]
🤔

File: saseumn/views/users.py [hunk 1]
```diff
@@ -159,6 +159,8 @@ def resumes(id=None):
         if not os.path.exists(path):
             return abort(404)
         return send_file(path, attachment_filename=resume.name)
+    if request.method == "POST":
+        db.session.query(Resume).filter(Resume.id==request.form.get("delete")).delete()
```
Comment (mzhang28) [inline saseumn/views/users.py:163]: shouldn't this use the `DELETE` http method instead of `POST` if ur deleting resumes

Comment (Crystal-Luo) [reply saseumn/views/users.py:163]: Apparently you can't use DELETE without Javascript? And I'm too lazy lmaoooo

**Comment by Crystal-Luo** [comment]
I'm not authorized to merge T_T

**Pull request merged**